docs/getting-started/getting-started-with-cwl.rst

```diff
@@ -237,7 +237,7 @@ how to run your tool.
 The first thing I'll do is
 `setup the Dockstore CLI locally <https://dockstore.org/quick-start>`__.
 This will have me install all of the dependencies needed to run the
-Dockstore CLI on my local machine.
+Dockstore CLI on my local machine. Make sure to install cwltool as well.
```

Comment:
This is beyond the scope of this PR, and maybe you tackled in your docs strike team, but I can't seem to find instructions on how to install cwltool in the docs (it is on the Getting Started of Dockstore proper). There could be a link here to it.

Reply:
The link above goes to the Dockstore Quick start page which has a cwltool installation guide.